Trying to get stats when database size is too big ends in a Serialization error

Hello,
I tried to get the stats of my meilisearch DB and it ended with a serialization error because database size is more than MAX_INT.
`com.meilisearch.sdk.api.instance.Stats.java `object has `private int databaseSize`
And if DB is larger than 2^31-1 = 2147483647 then it fails to serialize.
My DB is more than 7GB (7701901318) , so it fails.

```
Caused by: com.google.gson.JsonSyntaxException: java.lang.NumberFormatException: Expected an int but was 7701901318 at line 1 column 27 path $.databaseSize
	at com.google.gson.internal.bind.TypeAdapters$7.read(TypeAdapters.java:228)
	at com.google.gson.internal.bind.TypeAdapters$7.read(TypeAdapters.java:218)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:131)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:222)
	at com.google.gson.Gson.fromJson(Gson.java:932)
	at com.google.gson.Gson.fromJson(Gson.java:897)
	at com.google.gson.Gson.fromJson(Gson.java:846)
	at com.google.gson.Gson.fromJson(Gson.java:817)
	at com.meilisearch.sdk.json.GsonJsonHandler.decode(GsonJsonHandler.java:42)
```

--- a/src/main/java/com/meilisearch/sdk/api/instance/IndexStats.java
+++ b/src/main/java/com/meilisearch/sdk/api/instance/IndexStats.java
@@ -3,24 +3,24 @@ package com.meilisearch.sdk.api.instance;
 import java.util.Map;
 
 public class IndexStats {
-    private int numberOfDocuments;
+    private long numberOfDocuments;
     private boolean isIndexing;
     private Map<String, Integer> fieldDistribution;
 
     public IndexStats() {}
 
     public IndexStats(
-            int numberOfDocuments, boolean isIndexing, Map<String, Integer> fieldDistribution) {
+            long numberOfDocuments, boolean isIndexing, Map<String, Integer> fieldDistribution) {
         this.numberOfDocuments = numberOfDocuments;
         this.isIndexing = isIndexing;
         this.fieldDistribution = fieldDistribution;
     }
 
-    public int getNumberOfDocuments() {
+    public long getNumberOfDocuments() {
         return numberOfDocuments;
     }
 
-    public void setNumberOfDocuments(int numberOfDocuments) {
+    public void setNumberOfDocuments(long numberOfDocuments) {
         this.numberOfDocuments = numberOfDocuments;
     }
 

--- a/src/main/java/com/meilisearch/sdk/api/instance/Stats.java
+++ b/src/main/java/com/meilisearch/sdk/api/instance/Stats.java
@@ -4,23 +4,23 @@ import java.util.Date;
 import java.util.Map;
 
 public class Stats {
-    private int databaseSize;
+    private long databaseSize;
     private Date lastUpdate;
     private Map<String, IndexStats> indexes;
 
     public Stats() {}
 
-    public Stats(int databaseSize, Date lastUpdate, Map<String, IndexStats> indexes) {
+    public Stats(long databaseSize, Date lastUpdate, Map<String, IndexStats> indexes) {
         this.databaseSize = databaseSize;
         this.lastUpdate = lastUpdate;
         this.indexes = indexes;
     }
 
-    public int getDatabaseSize() {
+    public long getDatabaseSize() {
         return databaseSize;
     }
 
-    public void setDatabaseSize(int databaseSize) {
+    public void setDatabaseSize(long databaseSize) {
         this.databaseSize = databaseSize;
     }
 

--- a/src/test/java/com/meilisearch/sdk/api/instance/InstanceHandlerTest.java
+++ b/src/test/java/com/meilisearch/sdk/api/instance/InstanceHandlerTest.java
@@ -82,14 +82,14 @@ class InstanceHandlerTest {
                                 new BasicHttpResponse(
                                         null,
                                         200,
-                                        "{\"databaseSize\": 447819776,\"lastUpdate\": \"2019-11-15T11:15:22.092896Z\",\"indexes\": {\"movies\": {\"numberOfDocuments\": 19654,\"isIndexing\": false,\"fieldDistribution\": {\"poster\": 19654,\"overview\": 19654,\"title\": 19654,\"id\": 19654,\"release_date\": 19654}},\"rangemovies\": {\"numberOfDocuments\": 19654,\"isIndexing\": false,\"fieldDistribution\": {\"overview\": 19654,\"id\": 19654,\"title\": 19654}}}}"))
+                                        "{\"databaseSize\": 7701901318,\"lastUpdate\": \"2019-11-15T11:15:22.092896Z\",\"indexes\": {\"movies\": {\"numberOfDocuments\": 19654,\"isIndexing\": false,\"fieldDistribution\": {\"poster\": 19654,\"overview\": 19654,\"title\": 19654,\"id\": 19654,\"release_date\": 19654}},\"rangemovies\": {\"numberOfDocuments\": 19654,\"isIndexing\": false,\"fieldDistribution\": {\"overview\": 19654,\"id\": 19654,\"title\": 19654}}}}"))
                 .thenThrow(MeiliSearchRuntimeException.class);
         Stats stats = classToTest.getStats();
-        assertThat(stats.getDatabaseSize(), is(447819776));
+        assertThat(stats.getDatabaseSize(), is(7701901318L));
         assertThat(stats.getLastUpdate().toInstant().toEpochMilli(), is(1573816522092L));
         assertThat(stats.getIndexes().keySet(), hasItems("movies", "rangemovies"));
         IndexStats movies = stats.getIndexes().get("movies");
-        assertThat(movies.getNumberOfDocuments(), is(19654));
+        assertThat(movies.getNumberOfDocuments(), is(19654L));
         assertThat(movies.isIndexing(), is(false));
         assertThat(movies.getFieldDistribution().keySet(), hasItems("overview", "id", "title"));
         assertThat(movies.getFieldDistribution().get("overview"), is(19654));
@@ -108,7 +108,7 @@ class InstanceHandlerTest {
                                         "{\"numberOfDocuments\": 19654,\"isIndexing\": false,\"fieldDistribution\": {\"poster\": 19654,\"release_date\": 19654,\"title\": 19654,\"id\": 19654,\"overview\": 19654}}"))
                 .thenThrow(MeiliSearchRuntimeException.class);
         IndexStats stats = classToTest.getStats("index");
-        assertThat(stats.getNumberOfDocuments(), is(19654));
+        assertThat(stats.getNumberOfDocuments(), is(19654L));
         assertThat(stats.isIndexing(), is(false));
         assertThat(
                 stats.getFieldDistribution().keySet(),

